developer-docs/server-installation/core-services.md

```diff
@@ -53,39 +55,41 @@ Ensure that all Artifacts are uploaded
-| (Provision) Cassandra |Provisions Cassandra and create keyspaces required for Sunbird Core | Provisions Cassandra and creates keyspaces and performs migration | release-3.1.0|
-| Cassandra | Performs keyspace schema migration | Performs migration if required. Deploy this thrice by choosing different zip files using the build_number parameter. Ensure that you get a success message for the Cassandra migration on the Jenkins console output. Do not rely only on the red or green status indicator on Jenkins job | release-3.1.0
-|(Provision) Keycloak | Install dependencies for keycloak | Provisions Keycloak by installing prerequisites like Java and environment variables | release-3.1.0
-| Keycloak | Deploys Keycloak service to VM | Centralized user management for sunbird | release-3.1.0
-| KeycloakRealm(Core folder) |  User management -Creates sunbird realm | Creates a Sunbird Realm. After the Sunbird realm is created, configure Keycloak by using the steps mentioned in the **Keycloak Configuration** section | release-3.1.0
-| Player |  UI for sunbird  | Deploys portal UI | release-3.1.0
-| Learner | Deploys the Learner Service |  Handles user management and helps to search org | release-3.1.0
-| Content | Deploys content service | Helps to create content | release-3.1.0
-| Search              | Deploys the search service artefacts and starts the search service | |release-3.1.0
-| KnowledgeMW | Deploys knowledgemw service | Deploys the knowledgemw service | release-3.1.0
-| Lms | Deploys LMS Service | Provides the APIs for LMS functionality of Sunbird| release-3.1.0
-| certTemplate | Deploys template required for signing certificate | Creates a sample template for generating certificates | release-3.1.0
-| EncService | Deploys enc service | Encrypts and decrypt the keys to generate certificate | release-3.1.0
+| (Provision) Cassandra |Provisions Cassandra and create keyspaces required for Sunbird Core | Provisions Cassandra and creates keyspaces and performs migration | release-3.4.0|
+| Cassandra | Performs keyspace schema migration | Performs migration if required. Deploy this thrice by choosing different zip files using the build_number parameter. Ensure that you get a success message for the Cassandra migration on the Jenkins console output. Do not rely only on the red or green status indicator on Jenkins job | release-3.4.0
+|(Provision) Keycloak | Install dependencies for keycloak | Provisions Keycloak by installing prerequisites like Java and environment variables | release-3.4.0
+| Keycloak | Deploys Keycloak service to VM | Centralized user management for sunbird | release-3.4.0
+| KeycloakRealm(Core folder) |  User management -Creates sunbird realm | Creates a Sunbird Realm. After the Sunbird realm is created, configure Keycloak by using the steps mentioned in the **Keycloak Configuration** section | release-3.4.0
+|Adminutil | Deploys the Adminutil container | Creates tokens for the sunbird devices | release-3.0.0
```

Comment:
@G33tha should this be 3.0.0 or 3.4.0?

Reply:
@Sunitashelar it has to be 3.4.0, i have updated the version, Please check